lib/vanagon/platform/rpm/sles.rb

```diff
@@ -10,21 +10,21 @@ class SLES < Vanagon::Platform::RPM
         # @return [Array] A list of commands to add a zypper repo for the build system
         def add_repository(definition)
           definition = URI.parse(definition)
-          if @os_version == '10'
-            flag = 'sa'
-          else
-            flag = 'ar'
-          end
+          flag = if @os_version == '10'
```

Comment:
I left some notes about this but I am strongly in favor of inline disabled Cops.